src/GradedTestSets.jl

```diff
@@ -8,7 +8,7 @@ using Test: get_testset_depth, get_testset
 
 mutable struct GradedTestSet <: AbstractTestSet
     description::String
-    points::Int # points scored if all sub-tests pass
+    points::Real # points scored if all sub-tests pass
```

Comment:
Field types should not be abstract. If this can't be made a parametric type, maybe `Float64` is a better choice?

Reply:
I was assuming you wanted `Real` due to the prior message. `Float64` is working as well since I doubt anyone would try to use other number types such as complex numbers for grading purposes. Will change everything to `Float64`